### ISSUE TITLE
start a userguide

### ISSUE THREAD

**Issue by stapelberg** [gokrazy/gokrazy#51]
To explain the following features:

* [ ] remote syslog

**Comment by prologic** [comment]
I was going to suggest; why not just use the built-in Wiki? I'd even be happy to contribute; although I haven't used gokrazy all that much to be honest (yet)

**Comment by stapelberg** [comment]
I’ll think about it. In general, I like having the documentation on the proper website, but perhaps the wiki is a more pragmatic solution.

**Comment by prologic** [comment]
Yeah I tried that once myself with one of my projects with the whole Github Pages thing and found it too much work to maintain and get it looking nice. The Builtin Wiki is definately more convenient I think in this case plus you can contribute to it like any other repo.

**Comment by stapelberg** [comment]
I’ve come to really like the editing experience with https://gohugo.io/, and have used it successfully for https://distr1.org/.

The distri website sources are located in https://github.com/distr1/distri/tree/master/website and instruct hugo to render to the https://github.com/distr1/distri/tree/master/docs subdirectory, which is then picked up by GitHub pages.

I think we can do the same thing here, i.e. convert https://github.com/gokrazy/gokrazy.github.io to a hugo instance which renders to the docs/ subdirectory and then serve that. Shouldn’t be too involved.

**Comment by stapelberg** [comment]
Okay, the userguide is now started at https://gokrazy.org/userguide/, and contributions of the remaining topics (see first comment of this issue) are welcome! Please comment here when you start working on a topic so that others don’t accidentally duplicate your work.
